From a5a47bf2816e05f823216833b402633fd5ee0d6b Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 14 Aug 2019 11:56:36 +0300
Subject: [PATCH] Revert "[SW-0] gaudi: fix compilation erroe"

It turns out the original change (which is fixed by this commit) is not
needed, so we revert both commits.
Here revert the fix.

This reverts commit 73ea388abbbd46381469edc3dbf8543e8df2d0cc.

Change-Id: I47f339712c2afaed109a08caf44d7e082dd76dfe
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/asic.c b/gaudi/asic.c
index 87785f1..c760a0d 100644
--- a/gaudi/asic.c
+++ b/gaudi/asic.c
@@ -298,7 +298,7 @@
 u32 hl_skip_pcie_init(void)
 {
 	/* Use DumpSel[11] to skip PCIe initialization (active low) */
-	return !(regmap_readl(mmPSOC_GLOBAL_CONF_BOOT_STRAP_PINS) & BIT(19));
+	return !regmap_readl(mmPSOC_GLOBAL_CONF_BOOT_STRAP_PINS) & BIT(19);
 }
 
 u64 hl_pll_get_cpu_cycles_per_usec(void)
